sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1284,6 +1327,13 @@ def task_worker(self):
                 try:
                     # CMIS state transitions
                     if state == self.CMIS_STATE_INSERTED:
+                        if self.port_dict[lport]['host_tx_ready'] != 'True' or \
```

Comment:
Please change it 'true' and 'false' <small case> for host_tx_ready handling through out the file as we discussed.